script.sh

```diff
@@ -269,7 +269,7 @@ docker cp "$(cat ${CIDFILE}):${TRAVIS_DEBIAN_BUILD_DIR}"/ - \
 
 if [ "${TRAVIS_DEBIAN_AUTOPKGTEST}" = "true" ]
 then
-	docker run --cidfile=${CIDFILE} --interactive ${TAG} /bin/sh - <<EOF
+	docker run --volume $(readlink -f ${TRAVIS_DEBIAN_TARGET_DIR}):${TRAVIS_DEBIAN_BUILD_DIR} --interactive ${TAG} /bin/sh - <<EOF
```

Comment:
`--volume` needs either a volume name or a absolute path on the host, see https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume

`TRAVIS_DEBIAN_TARGET_DIR` defaults to `../` which is relative :)